src/app/core/entity-components/entity-subrecord/entity-subrecord.component.ts

```diff
@@ -106,6 +111,10 @@ export class EntitySubrecordComponent<T extends Entity>
   private screenWidth = "";
 
   @ViewChild(MatSort) sort: MatSort;
+  @ViewChild(MatPaginator) paginator: MatPaginator;
+
+  paginatorPageSize = 10;
+  paginatorPageIndex = 0;
```

Comment:
as we don't save the index, it is not necessary to track and update it explicitly here in the component